internal/model/rh_account.go

```diff
@@ -6,7 +6,7 @@ import (
 
 type RHAccount struct {
 	ID      uint   `gorm:"primaryKey;not null;autoIncrement"`
-	Account string `gorm:"type:text;unique"`
+	Account string `gorm:"type:text;"`
```

Comment:
I just confirmed we don't consider both account and orgid while looking up for the rh_accounts, I can see we lookup based on orgId, so we are good even when account is sometimes empty and sometimes having some value,
~~~
rh_account.go:	
result := db.Where("org_id = ?", r.OrgId).FirstOrCreate(r)
~~~

Reply:
Yes, That's correct